<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAlarmWindow</class>
 <widget class="QDialog" name="AddAlarmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>223</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>223</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(151, 151, 151);
item {
background-color: transparent;
}
item:selected {
background-color: rgb(163, 3, 47);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 10pt;
    color: black;
    font: bold;</string>
      </property>
      <property name="text">
       <string>Назва: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 10pt;
    color: black;
    font: bold;</string>
      </property>
      <property name="text">
       <string>Задати час:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 10pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
      </property>
      <property name="displayFormat">
       <string>HH:mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
   </property>
   <property name="text">
    <string>Текст по закінченню</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>182</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(46, 103, 156); 
    border-radius: 3px; 
    padding-right: 10px; 
    padding-left: 10px;
    padding-top: 5px;
    padding-bottom: 5px;
    background-color: rgb(77, 138, 201);
    color: white;
    font: bold;
    width: 64px;</string>
   </property>
   <property name="text">
    <string>Запустити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>182</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
   </property>
   <property name="text">
    <string>Відхилити</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
   </property>
   <property name="text">
    <string>Не турбувати</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>162</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>144444</width>
     <height>144444</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 7pt;
    color: red;
    font: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>34</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 7pt;
    color: red;
    font: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>71</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 7pt;
    color: red;
    font: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
